core/services/user_service/controllers/auth_controller.py

```diff
@@ -48,18 +48,20 @@ def login(email: str, password: str) -> Tuple[str, str]:
     return access_token, refresh_token
 
 
-def refresh_access_token(refresh_token: str):
+def refresh_access_token(access_token: str, refresh_token: str):
     """
     The refresh function takes a refresh token
     and generates a new access_token.
     Returns a tuple of access_token and refresh_token
     """
-    payload = verify_token(refresh_token)
-
-    if not payload:
+    payload_refresh = verify_token(refresh_token)
+    if not payload_refresh:
         raise HTTPException(status_code=401, detail="Invalid refresh token")
 
-    email: str = payload.get("email")
+    # using old access token to get user email
+    payload_access = verify_token(access_token)
```

Comment:
Rename

Reply:
I will implement both a decode_token, which returns the payload and a verify_token which returns a boolean